proxies/aareg-proxy/src/main/java/no/nav/testnav/proxies/aaregproxy/AaregProxyApplicationStarter.java

```diff
@@ -18,69 +18,82 @@
 import org.springframework.context.annotation.Import;
 
 import java.util.function.Function;
+import java.util.stream.Stream;
 
 @Import({
-        CoreConfig.class,
-        DevConfig.class,
-        SecurityConfig.class
+    CoreConfig.class,
+    DevConfig.class,
+    SecurityConfig.class
 })
 @SpringBootApplication
 public class AaregProxyApplicationStarter {
 
+    @Value("${app.modapp.uri.pattern:https://modapp-{{ENV}}.adeo.no}")
+    private String modappUriPattern;
+
     public static void main(String[] args) {
         SpringApplication.run(AaregProxyApplicationStarter.class, args);
     }
 
     @Bean
     public StsOidcTokenService stsPreprodOidcTokenService(
-            @Value("${sts.preprod.token.provider.url}") String url,
-            @Value("${sts.preprod.token.provider.username}") String username,
-            @Value("${sts.preprod.token.provider.password}") String password
+        @Value("${sts.preprod.token.provider.url}") String url,
+        @Value("${sts.preprod.token.provider.username}") String username,
+        @Value("${sts.preprod.token.provider.password}") String password
     ) {
         return new StsOidcTokenService(url, username, password);
     }
 
     @Bean
     public StsOidcTokenService stsTestOidcTokenService(
-            @Value("${sts.test.token.provider.url}") String url,
-            @Value("${sts.test.token.provider.username}") String username,
-            @Value("${sts.test.token.provider.password}") String password
+        @Value("${sts.test.token.provider.url}") String url,
+        @Value("${sts.test.token.provider.username}") String username,
+        @Value("${sts.test.token.provider.password}") String password
     ) {
         return new StsOidcTokenService(url, username, password);
     }
 
     @Bean
     public RouteLocator customRouteLocator(RouteLocatorBuilder builder, StsOidcTokenService stsTestOidcTokenService, StsOidcTokenService stsPreprodOidcTokenService) {
+        RouteLocatorBuilder.Builder routes = builder.routes();
 
         var preprodFilter = AddAuthenticationRequestGatewayFilterFactory
-                .bearerAuthenticationAndNavConsumerTokenHeaderFilter(stsPreprodOidcTokenService::getToken);
+            .bearerAuthenticationAndNavConsumerTokenHeaderFilter(stsPreprodOidcTokenService::getToken);
+        Stream.of("q0", "q1", "q2", "q4", "q5", "qx")
```

Comment:
Q0 er prodlikt miljø som vi ikke skal støtte mer.

Reply:
Fjernet støtte for Q0 (og T6).